server/api/store.go

```diff
@@ -77,10 +129,7 @@ const (
 
 func newStoreInfo(opt *config.ScheduleConfig, store *core.StoreInfo) *StoreInfo {
 	s := &StoreInfo{
-		Store: &MetaStore{
-			Store:     store.GetMeta(),
```

Comment:
Why not just copy a `metapb.Store` here and set the `State` field to empty?

Reply:
I don't think it is safe to change the value of the field itself, as if there is any other assignment action after the function `newStoreInfo` executed, it will result in incorrect results. For example, render.go assigns the value of `StoreInfo` to `trendStore`

Reply:
OK, I mistakenly thought that this structure would only be used to be displayed.